.eslintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "./node_modules/gts/"
+  "extends": "./node_modules/gts/",
+  "parserOptions": {
+    "ecmaVersion": 2020
+  }
```

Comment:
What does this do?

Reply:
This needs to be specified to support the `import.meta` call here: https://github.com/sass/sass/pull/3112/files#diff-fe16e804620b4c53d6bcf6fbfe0e6580e9b2b3a89f05f120ccf914e971238884R31

Reply:
Looks like the `import.meta` wasn't even necessary, so I removed it.